test/expectations/expectations_test_runner.rb

```diff
@@ -34,6 +34,9 @@ def default_args
         include ExpectationsRunnerMethods
       RB
 
+      # The fixtures tests take some time to run, this allows us to skip them locally if desired
+      return if ENV["RUBY_LSP_SKIP_FIXTURES_TEST"]
```

Comment:
Useful? I can move to a separate PR if desired.

Reply:
Out of curiosity, what tests made you want to add this? I assume you ran `RUBY_LSP_SKIP_FIXTURES_TEST=true dev test`, but what tests were in your mind when running it?

Asking because it may be a potential opportunity to add ways to run tests. For example, maybe we can make the `Run Tests` option work when right-clicking on a folder? It'll make running all ruby indexer's tests easier.

<img width="381" alt="Screenshot 2024-03-05 at 19 37 14" src="https://github.com/Shopify/ruby-lsp/assets/5079556/ba3cd834-21d1-485e-8fcc-83da708206a5">